charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -193,18 +199,22 @@ data:
         # This is needed because the kubelets' certificates are not are generated
         # for a specific pod IP
         insecure_skip_verify: true
-        cert_file: /srv/kubernetes/prometheus-kubelet/prometheus-kubelet.crt
-        key_file: /srv/kubernetes/prometheus-kubelet/prometheus-kubelet.key
+        cert_file: /etc/prometheus/seed/prometheus.crt
+        key_file: /etc/prometheus/seed/prometheus.key
       kubernetes_sd_configs:
       - role: node
         api_server: https://kube-apiserver:443
         tls_config:
 {{ include "prometheus.tls-config.kube-cert-auth" . | indent 10 }}
       relabel_configs:
-      - source_labels: [ __meta_kubernetes_node_address_InternalIP ]
-        target_label: instance
       - action: labelmap
         regex: __meta_kubernetes_node_label_(.+)
+      - target_label: __address__
```

Comment:
This isn't related to this PR. It seems to be because we are using a rate of 1m and the scrape interval is also 1m. I will increase the rate to 2m in a separate PR.